pageserver/src/page_service.rs

```diff
@@ -496,8 +489,11 @@ impl postgres_backend::Handler for PageServerHandler {
 
             // Check that the timeline exists
             let repository = page_cache::get_repository_for_tenant(&tenantid)?;
-            if repository.get_timeline(timelineid).is_err() {
-                bail!("client requested callmemaybe on timeline {} which does not exist in page server", timelineid);
+            if let Err(e) = repository
```

Comment:
This could also use ? like the other calls:
```
            // Check that the timeline exists
            let repository = page_cache::get_repository_for_tenant(&tenantid)?;
            repository
                .get_timeline(timelineid)
                .context(format!("error fetching timeline {}", timelineid))?;
```

Reply:
We don't need the variable, so have to get warning or prefix it with underscore, that's not perfect as well.

Reply:
Indeed.